modules/nw-infw-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-infw-operator-troubleshooting_{context}"]
+= Troubleshooting Ingress Node Firewall Operator
+
+* Run the following command to collect all ingress firewall node pods logs:
+
+[source,terminal]
+----
+oc adm must-gather – gather_ingress_node_firewall
+----
+
+The logs are available in the sos node's report containing eBPF bpftool outputs at: `/sos_commands/ebpf`. These reports include lookup tables used or updated as the Ingress Node Firewall Operator processes the packets.
```

Comment:
Ingress firewall operator just the configuration plane , ingress firewall XDP is the one that handle packet processing, update statistics and emit events

Reply:
addressed